src/test/mod.rs

```diff
@@ -229,6 +230,14 @@ pub fn fixture_repo(provider: FixtureProvider) -> io::Result<TempDir> {
 
             Ok(path)
         }
+        FixtureProvider::GitBare => {
+            let path = tempfile::tempdir()?;
+            create_command("git")?
+                .current_dir(path.path())
+                .args(["--bare", "init"])
+                .output()?;
```

Comment:
I had a look at switching to use `gix::ThreadSafeRepository::init` but ran in to some challenges due to the expected return type: https://github.com/starship/starship/blob/4d575b62745c06cecf911c63b1e11521739bdb4c/src/test/mod.rs#L174

So I'm not sure how easy it would be to utilise `gix::ThreadSafeRepository::init` given the above.
In addition I've followed the convention of the other fixture providers e.g:

https://github.com/starship/starship/blob/4d575b62745c06cecf911c63b1e11521739bdb4c/src/test/mod.rs#L191-L199

Let me know what you think?

Reply:
Switched to `gix::ThreadSafeRepository::init`.